README.md

```diff
@@ -1,7 +1,35 @@
-# Settings View package [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
+# Settings View [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view) [![Dependency Status](https://david-dm.org/atom/settings-view.svg)](https://david-dm.org/atom/atom)
 
-Edit config settings, install packages, and change themes in Atom.
+Edit core config settings, install and configure packages, and change themes all from within Atom.
 
-Open by pressing `cmd-,` or selecting the _Atom > Preferences..._ menu.
+![Settings View](https://cloud.githubusercontent.com/assets/1476/5624241/d29ffbe6-9516-11e4-8bfe-e91623977bc7.png)
 
-![screen shot 2015-01-05 at 8 08 18 pm](https://cloud.githubusercontent.com/assets/1476/5624241/d29ffbe6-9516-11e4-8bfe-e91623977bc7.png)
+## Usage
+You can open the Settings View by navigating to _Atom > Preferences_ (Mac), _File > Preferences_ (Windows), or _Edit > Preferences_ (Linux).
+
+In order to install new packages and themes, click on the _Install_ section on the left-hand side.
+Once installed, community packages/themes (and their settings) are housed within their respective section.
+All packages/themes that have updates will be listed under the _Updates_ section.  Finally, all keybindings (including ones that community packages have added) are available in the _Keybindings_ section.
```

Comment:
I noticed that there's a double space here (before "Finally"), but then also noticed that you have each sentence in a separate line, so you probably wanted to do that here as well?